frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -223,19 +225,36 @@ export const FeatureStrategyMenu = ({
                 PaperProps={{
                     sx: (theme) => ({
                         paddingBottom: theme.spacing(1),
+                        width: 'auto',
+                        maxWidth: '95vw',
+                        overflow: 'hidden',
```

Comment:
Will this also activate on the old design? I'm assuming it doesn't change anything.